.github/workflows/nomad-common.yaml

```
@@ -374,6 +369,35 @@ jobs:
+    if: always()
+    steps:
+      - uses: actions/checkout@v6
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@v31
+
+      - name: Setup cachix
+        uses: cachix/cachix-action@v16
+        with:
+          name: holochain-ci
+          authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
```

actionlint (1.7.9)

property "cachix_auth_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; hetzner_holochain_infra_buckets_access: string; hetzner_holochain_infra_buckets_secret: string; influx_token: string; nomad_access_token: string}

(expression)